commands.json

```diff
@@ -3912,6 +3912,22 @@
           "SAVE"
         ],
         "optional": true
+      },
+      {
+        "name": "now",
```

Comment:
IIUC, this file is generated from redis and I should not touch this file. I'll revert it. Is that right?

Reply:
As long as we're still using this version of the docs/redis-io (which doesn't use the generated file from the redis repo) we should keep both the redis repo and the redis-doc repo manually in sycn. So don't revert.